moveit_msgs/AttachedCollisionObject[] attached_objects
moveit_msgs/CollisionObject[] collision_objects
---
bool success
